src/collections.ts

```diff
@@ -0,0 +1,147 @@
+import * as collections from "@opencreek/deno-std-collections"
+import { apply, extendProtoype } from "./extendPrototype"
+
+type Collections = typeof collections
+
+function filterNotNullish<T>(this: Array<T>): Array<NonNullable<T>> {
+  return this.filter((it): it is NonNullable<T> => it != undefined)
```

Comment:
Nitpick: The agreed standard to do that exception to the `==` vs `===` rule is to use `null` instead of `undefined` (even though both work), see [the respective ESLint rule](https://eslint.org/docs/rules/eqeqeq)